diff --git a/src/ath.h b/src/ath.h
index 8243864a..1b3f73f7 100644
--- a/src/ath.h
+++ b/src/ath.h
@@ -24,6 +24,7 @@
 #ifdef HAVE_STDINT_H
 #include <stdint.h>
 #endif
+
 #ifdef HAVE_W32_SYSTEM
   /* fixme: Check how we did it in libgcrypt.  */
   struct msghdr { int dummy; };
@@ -34,6 +35,9 @@
 #else /*!HAVE_W32_SYSTEM*/
 # ifdef HAVE_POLL_H
 #  include <poll.h>
+#  if defined (__MVS__) && defined (HAVE_SYS_TIME_H)
+#    include <sys/time.h>
+#  endif
 # else
 #  ifdef HAVE_SYS_SELECT_H
 #   include <sys/select.h>
